index.html

```diff
@@ -43,30 +43,33 @@
 
     <div class="top-div">
       <div class="container pb-3">
-        <ul class="list-unstyled">
-          <li>
-            <i class="fas fa-map-marker-alt text-secondary"></i>
-            <a href="https://yandex.ru/maps/org/dve_pyatyorki/1024420856/?ll=20.509270%2C54.727694&oid=1024420856&ol=biz&z=18">
+        <div class="row">
+          <ul class="list-unstyled col-sm-6">
```

Comment:
Please could you wrap ul and img tags in div with col-sm-6 class. Semantically it looks better and more flexible than it is now.  For example in case we need to put something in columns you can add element directly in parent div.

Reply:
added parent div elements for ul and img